engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -95,6 +99,8 @@ object WorkflowExecutionActor {
     override val exceptionContext = s"WorkflowExecutionActor"
   }
 
+  val MaxRetries = ConfigFactory.load().getInt("system.maxRetries")
```

Comment:
@cjllanwarne in conjunction with your earlier observation that this is more appropriately a backend property, I might suggest having the actor factory sanity check this on startup for fail fast.  I had similar concerns in this forthcoming JES PR.